src/EnergyPlus/DataDaylighting.hh

```diff
@@ -95,88 +96,89 @@ namespace DataDaylighting {
         DElightDaylighting
     };
 
-    // Parameters for "Lighting Control Type"
-    enum class iLtgCtrlType
+    // Parameters for "Lighting Control Type" - these are the values expected by DElight
+    enum class LtgCtrlType
     {
+        Invalid = 0,
         Continuous = 1,
         Stepped = 2,
-        ContinuousOff = 3
+        ContinuousOff = 3,
     };
 
-    struct IntWinAdjZoneExtWinStruct // nested structure for ZoneDaylight
+    constexpr std::array<std::string_view, 4> LtgCtrlTypeNamesUC = {"INVALID", "CONTINUOUS", "STEPPED", "CONTINUOUSOFF"};
+
+    struct IntWinAdjEnclExtWinStruct // nested structure for EnclDaylightCalc
     {
         // Members
-        int SurfNum;           // exterior window index
-        int NumOfIntWindows;   // count of interior windows associated with this ext win
-        Array1D_int IntWinNum; // index numbers for interior windows assoc with this ext win
-
-        // Default Constructor
-        IntWinAdjZoneExtWinStruct() : SurfNum(0), NumOfIntWindows(0)
-        {
-        }
+        int SurfNum = 0;         // exterior window index
+        int NumOfIntWindows = 0; // count of interior windows associated with this ext win
+        Array1D_int IntWinNum;   // index numbers for interior windows assoc with this ext win
     };
 
     struct EnclDaylightCalc
     {
         Real64 aveVisDiffReflect = 0.0; // Area-weighted average inside surface visible reflectance of zone
         Real64 totInsSurfArea = 0.0;    // Total inside surface area of a daylit zone (m2)
         Real64 floorVisRefl = 0.0;      // Area-weighted visible reflectance of floor of a daylit zone
+        int TotalExtWindows = 0;        // Total number of exterior windows in the zone or same solar enclosure
+        Array1D_int AdjIntWinEnclNums;  // List of enclosure numbers of adjacent enclosures that have exterior windows and
+        // share one or more interior windows with target enclosure
+        int NumOfIntWinAdjEncls = 0; // Number of adjacent enclosures that have exterior windows and share one or
```

Comment:
Same comment as above. Is a separate `NumOfX` variable needed when it is redundant with `X.size()`?